pkg/tsdb/grafana-pyroscope-datasource/service.go

```diff
@@ -55,50 +60,71 @@ func newInstanceSettings(httpClientProvider httpclient.Provider, ac accesscontro
 }
 
 func (s *Service) QueryData(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
+	logger.Debug("Received query request", "queries", req.Queries)
+
 	i, err := s.getInstance(ctx, req.PluginContext)
 	if err != nil {
+		logger.Debug("Failure in processing query request", "error", err)
 		return nil, err
 	}
-	return i.QueryData(ctx, req)
+
+	response, err := i.QueryData(ctx, req)
+	logger.Debug("Successfully processed query request")
+	return response, err
 }
 
 func (s *Service) CallResource(ctx context.Context, req *backend.CallResourceRequest, sender backend.CallResourceResponseSender) error {
+	logger.Debug("Successfully processed query request")
 	i, err := s.getInstance(ctx, req.PluginContext)
 	if err != nil {
+		logger.Debug("Successfully processed query request")
 		return err
 	}
+	logger.Debug("Successfully processed query request")
```

Comment:
Why are we logging the same message? Can we update the error one on line 80. Also, could remove the one on line 77 and use line 83 to indicate success.